src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +25,21 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public static String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        String fileText;
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fileName))) {
+            fileText = bufferedReader.lines()
```

Comment:
can we read each character from file separately in order not to convert to String and then back to char array?